copycat/runtime/src/main/java/org/apache/kafka/copycat/runtime/distributed/DistributedHerder.java

```diff
@@ -274,67 +285,77 @@ public void stop() {
     @Override
     public synchronized void addConnector(final Map<String, String> connectorProps,
```

Comment:
Add trace message? We are doing this in the other API calls and will be nice to be consistent.